examples/map_transformations/reprojection_different_observers.py

```diff
@@ -63,26 +63,19 @@
            ['Limb as seen by AIA', 'Limb as seen by EUVI A'])
 
 ######################################################################
-# We now need to construct an output WCS. We build a custom header using
-# :func:`sunpy.map.header_helper.make_fitswcs_header` but we use a lot of the ``map_aia``
-# properties to do it.
+# Due to a mismatch between the solar radius as defined in ``map_aia``
+# and ``map_euvi``, our reprojection will not behave correctly on pixels
+# very near the limb. This can be prevented by equating the two radii.
```

Comment:
```suggestion
# very near the limb. This can be prevented by assuming that the emission
# observed by EUVI was emitted at the same radius from the centre of the Sun
# as the emission observed by AIA.
```

Reply:
I think this is a bit clearer on exactly what the line of code means in terms of assumption

Reply:
I agree that it is clearer as to what an assumption about rsun means in general but I think there might be a danger there that readers leave with the idea that the mismatch between EUVI (or really the sunpy default rsun value) and AIA rsun is because of different emission heights.

Reply:
To make it all clear, how about:
> Data providers can set the radius at which emission in the map is assumed to have come from. Most maps set this to the standard photospheric radius (including EUI maps), but some maps (including AIA maps) set it to a slightly larger value. This mismatch means our reprojection won't work correctly on pixels near the limb. This can be prevented by assuming that the emission observed by EUVI was emitted at the same radius from the centre of the Sun as the emission observed by AIA.

? I think it's not a bad thing if users go away understanding that emission can come from different heights, and there has to be an assumption about those heights somewhere?

Reply:
I like this paragraph, I think I was being stupid: my thought was this needs an explanation page for what rsun is, how it can have different values and why that can cause problems with reproject.

Now I have realised that this text can go in here now and when the page is made, be removed or reduced in place of a link, will make the change.